File: tests/nodejs/comment/getComment.js
var request = require('request');

var jsonBody = {
	thread_id: 1
}

request({
	url: 'http://localhost:3000/getComment',
	method: 'post',
	json: true,
	body: jsonBody
}, function (error, response, body) {
	console.log(response);
});

File: tests/python/comment/getComment.py
import requests
import json
import sys

jsonBody = {
	"thread_id": 1,
}

headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
res = requests.post('http://localhost:3000/getComment', data = json.dumps(jsonBody), headers = headers)
print(res.json())